packages/main/src/SegmentedButtonItem.ts

```diff
@@ -0,0 +1,110 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+
+import type { I18nText } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import SegmentedButtonItemTemplate from "./generated/templates/SegmentedButtonItemTemplate.lit.js";
+
+import ToggleButton from "./ToggleButton.js";
+import ButtonDesign from "./types/ButtonDesign.js";
+import Icon from "./Icon.js";
+
+// @ts-ignore
```

Comment:
this @ts-ignore comment will be redundant when you rebase. See #6366

Reply:
fixed